google/cloud/internal/populate_rest_options.h

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_INTERNAL_POPULATE_REST_OPTIONS_H
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_INTERNAL_POPULATE_REST_OPTIONS_H
+
+#include "google/cloud/options.h"
+#include "google/cloud/version.h"
+
+namespace google {
+namespace cloud {
+GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
+namespace internal {
+
+/**
+ * Modify @p opts to have default values for REST options.
+ *
+ * It is expected that `EndpointOption` has already been set on `opts` by
+ * `PopulateCommonOptions`.
+ *
+ * @param opts the current options. Any values already present in this
+ *     collection are not modified.
```

Comment:
This doesn't appear to be true when a present `EndpointOption` doesn't start with "http".

Reply:
Fixed, thanks.